serverless/lambda/src/main/java/uk/gov/di/lambdas/VerifyCodeHandler.java

```diff
@@ -81,20 +82,32 @@ public APIGatewayProxyResponseEvent handleRequest(
                 case VERIFY_EMAIL:
                     if (codeStorageService.isCodeBlockedForSession(
                             session.get().getEmailAddress(), session.get().getSessionId())) {
+
+                        if (isInvalidUserJourneyTransition(
+                                session.get().getState(), EMAIL_CODE_MAX_RETRIES_REACHED)) {
```

Comment:
So this will break the front end which is fine, but currently a generic status is handled for when a code is reached its max retries do we want to reimplement that?

Reply:
As I understand, if the user legitimately reaches max retries reached (i.e. it can only come from the previous state of email code invalid) then it will "fall through" and still return the generic 200 with status code, and only drop out if it somehow jumps to max retries reached which, in practice, I don't think is possible because you need to attempt at least one code entry first?

Is that what you're asking?

Reply:
my bad i think i got confused with what the front end did i am going to close this convo.